.github/scripts/generate-crd-docs/generate-crd-docs.sh

```diff
@@ -63,6 +63,7 @@ for api_group in "$OPERATOR_API_ROOT"*; do
       --source-path="./$api_version" \
       --renderer="$RENDERER" \
       --config "$RENDERER_CONFIG_FILE" \
+      --max-depth 10 \
```

Comment:
just for curiosity: why you decided to select 10?

Reply:
The default value is 6, what was not enough for us anymore, therefore it needed to be increased and 10 is ok for now. It just says how deep should the auto-generation of CRDs go

Reply:
then let's add a comment that if we go to 11, we know where to touch :)

Reply:
We're at 7 currently (if I counted it right), so we have enough space to grow if needed. I will add a comment